@navikt/core/css/darkside/primitives/page.darkside.css

```diff
@@ -27,11 +24,11 @@
 }
 
 .navds-page__content--padding {
-  padding-block-end: var(--a-spacing-16);
+  padding-block-end: var(--ax-spacing-16);
 }
 
 .navds-pageblock--text {
-  max-width: calc(var(--a-text-width-max) + var(--__axc-page-padding-inline) + var(--__axc-page-padding-inline));
+  max-width: calc(576px + var(--__axc-page-padding-inline) + var(--__axc-page-padding-inline));
```

Comment:
I want to remove the `text-width-max`-token since i don't think it deserves a place in the token-package. It's so context-based related to gutters, spacing and type of application that it's hard to generalize to much. So yes it will be permanent 👍